package/MDAnalysis/analysis/density.py

```diff
@@ -457,7 +467,7 @@ def _prepare(self):
         self._edges = edges
         self._arange = arange
         self._bins = bins
-        self.density = None
+        self.results.density = None
```

Comment:
Should we also remove this line.`self.results.density` is defined in L489? Or leave it as a kind of comment?

Reply:
Yeah that's a good point, I guess it never really did anything. We might as well remove it.